Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "gcmalloc"
+version = "0.1.0"
+authors = ["Jacob Hughes <jacob.hughes@kcl.ac.uk>"]
```

Comment:
I suggest you use a non-KCL email address that you're more likely to monitor in the long term, but it's up to you.

Reply:
Good idea